packages/eslint-plugin/src/tests/no-duration-frames-infinity.test.ts

```diff
@@ -0,0 +1,47 @@
+import { ESLintUtils } from "@typescript-eslint/experimental-utils";
+import rule from "../rules//no-duration-frames-infinity";
+
+const ruleTester = new ESLintUtils.RuleTester({
+  parser: "@typescript-eslint/parser",
+  parserOptions: {
+    ecmaFeatures: {
+      jsx: true,
+    },
+  },
+});
+
+ruleTester.run("no-duration-frames-infinity", rule, {
+  valid: [
+    `
+      import {Sequence} from 'remotion';
+
+      export const Re = () => {
+        return (
+          <Sequence durationInFrames={1}>
+            Hi
+          </Sequence>
+        );
+      }
+    `,
+  ],
+  invalid: [
+    {
+      code: `
+        import {Composition} from 'remotion';
+
+        export const Re = () => {
+          return (
+            <Sequence>
```

Comment:
@JonnyBurger this has to be invalid when `durationInFrames={Infinity}`. That's reason build failed, I will push to update this.

Reply:
You fixed it closing

Reply:
Yep I fixed it! Now the ESLint rule will even suggest a fix 😎